extensions/interactions/TextInput/TextInput.html

```diff
@@ -1,7 +1,25 @@
 <style>
+  ::-webkit-input-placeholder {
+   font-style: italic;
+  }
+  :-moz-placeholder {
+   font-style: italic;  
+  }
+  ::-moz-placeholder {
+   font-style: italic;  
+  }
+  :-ms-input-placeholder {  
+   font-style: italic; 
+  }
+
   .text-input-container md-input-group.md-default-theme label {
     margin-bottom: 0;
   }
+
+  .form-control:focus {
```

Comment:
This is too general and will apply to all .form-controls on the entire page. Instead, give the input field a class like text-input-single-line-field, and use that as the selector.